client/src/components/Atoms/ListAndDetail.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { ReactNode } from 'react'
+import styled from 'styled-components'
+import { Button, Card, Classes, Colors, InputGroup } from '@blueprintjs/core'
+
+/**
+ * A set of components to render a list-and-detail panel
+ *
+ * Example usage:
+ *
+ *  <ListAndDetail>
+ *    <List>
+ *      <ListItem>Item 1</ListItem>
+ *      <ListItem selected>Item 2</ListItem>
+ *    </List>
+ *    <Detail>
+ *      {detailsForSelectedListItem}
+ *    </Detail>
+ *  </ListAndDetail>
+ */
+
+export const ListAndDetail = styled(Card)`
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  overflow-y: auto;
+  padding: 0;
+  width: 100%;
+`
+
+// ---------- List ----------
+
+const ListContainer = styled.div`
+  border-right: 1px solid ${Colors.LIGHT_GRAY2};
+`
+
+const ListSearch = styled(InputGroup).attrs({
+  leftIcon: 'search',
+  type: 'search',
+})`
+  margin: 16px 12px;
+`
+
+const ListItems = styled.ul<{ withTopBorder?: boolean }>`
+  ${props =>
+    props.withTopBorder && `border-top: 1px solid ${Colors.LIGHT_GRAY2};`}
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  padding-bottom: 40px;
+`
+
+interface IListProps {
+  search?: {
+    placeholder: string
+    setQuery: (query: string) => void
+  }
+}
+
+export const List: React.FC<IListProps> = ({ search, children }) => {
+  return (
+    <ListContainer>
+      {search && (
+        <ListSearch
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+            search.setQuery(e.target.value)
+          }
+          placeholder={search.placeholder}
+        />
+      )}
+      <ListItems withTopBorder={Boolean(search)}>{children}</ListItems>
+    </ListContainer>
+  )
+}
+
+// ---------- ListItem ----------
+
+const ListItemContainer = styled.li<{ selected?: boolean }>`
+  border-bottom: 1px solid ${Colors.LIGHT_GRAY4};
+
+  button.${Classes.BUTTON} {
```

Comment:
Could we change the background color of the button on hover to match the border defined above? I was noticing the slight mismatch of grays creating a bit of a weird look.

Reply:
https://github.com/votingworks/arlo/pull/1661/commits/7da851cd9e729e82afe9c00af758e647cd094fe9